vspace/src/main/java/edu/asu/diging/vspace/core/model/IDefaultExhibition.java

```diff
@@ -0,0 +1,14 @@
+package edu.asu.diging.vspace.core.model;
+
+import edu.asu.diging.vspace.core.model.impl.Space;
+
+public interface IDefaultExhibition extends IVSpaceElement {
```

Comment:
this should be called IExhibition (there can only be one default, but many exhibitions)